sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -31,6 +33,8 @@
 - Removed unused type `AccountResourceType` and `PathLeaseAction`.
 - Changed all previous `LeaseDuration` members to a new type named `LeaseDurationType`.
 - `startsOn` parameter for `GetUserDelegationKey` was changed to optional.
+- Changed `SetAccessControlRecursive` to `SetAccessControlRecursiveSinglePage`, to mark that it is a single page operation, and removed the `mode` parameter, separated the modify/delete functionality to two new APIs.
```

Comment:
You're also going to add `SinglePage` suffix to other page operations, right?

Do we have an issue tracking that already?

Reply:
Yes: https://github.com/Azure/azure-sdk-for-cpp/issues/1471